web/styles/drafts.css

```diff
@@ -1,103 +1,68 @@
-        }
-
-        .messagebox-content {
-            grid-template-rows: auto;
-            grid-template-columns: auto max-content;
-            padding: 10px;
-
-            .message_content {
-                grid-column: 1 / 2;
-                /* to space from restore draft button */
-                margin-right: 5px;
-            }
-
-            .message_top_line {
-                grid-column: 2 / 3;
+    .header-body {
+        display: flex;
+        align-items: center;
+        flex-direction: row;
+        justify-content: space-between;
+        gap: 5px;
+
+        .removed-drafts-message {
+            text-align: left;
+            margin-left: 25px;
+
+            @media (width < $lg_min) {
+                text-align: center;
+                margin-left: 0;
             }
         }
 
-        .messagebox {
-            cursor: auto;
-            box-shadow: none;
-            border-radius: 0 0 7px 7px;
-        }
-
-        .draft_controls {
-            display: inline-block;
-            font-size: 0.9em;
+        .delete-drafts-group {
+            display: flex;
+            justify-content: flex-end;
+            gap: 10px;
 
-            [data-tippy-root] {
-                width: max-content;
-                word-wrap: unset;
+            .delete-selected-drafts-button {
+                &:focus {
+                    background-color: hsl(0deg 0% 93%);
+                }
             }
 
-            .restore-draft {
-                cursor: pointer;
-                margin-right: 5px;
-                color: hsl(170deg 48% 54%);
-                opacity: 0.7;
+            .select-drafts-button {
+                display: flex;
+                align-items: center;
+                margin-right: 25px;
+                gap: 5px;
 
-                &:hover {
-                    opacity: 1;
+                &:focus {
+                    background-color: hsl(0deg 0% 93%);
                 }
             }
 
-            .delete-draft {
-                cursor: pointer;
-                margin-left: 5px;
-                color: hsl(357deg 52% 57%);
-                opacity: 0.7;
+            .select-state-indicator {
+                width: 15px;
+            }
 
-                &:hover {
-                    opacity: 1;
-                }
+            @media (width < $lg_min) {
+                margin-top: 5px;
+                width: 100%;
             }
         }
 
-        .message_header {
-            /* We don't need these effects applied for message list in the drafts overlay. */
-            box-shadow: none !important;
-            border: 0 !important;
-            background: transparent;
+        @media (width < $lg_min) {
+            flex-direction: column;
+        }
+    }
+
+    .drafts-list {
+        & h2 {
+            font-size: 1.1em;
+            line-height: normal;
+            margin-bottom: 5px;
         }
     }
+
+    .mark-draft {
+        margin-top: 5px;
+        width: 10px;
```

Comment:
I don't think you need this width here; testing this on a related PR this morning, that knocks the mark-draft icon out of alignment with the rest of the grid. Try removing and see if it's okay?

Reply:
Oooh, yeah, that's not good.

If you render the grid lines around the area though, on `.messagebox-content`, you'll see that the checkbox winds up too far to the right of the lines. So some other fix that keeps those boxes in line AND prevents the shift with the layout buttons is what's called for here.

Reply:
I have changed the width to `15px` and then now the checkbox is inside the grid line. I also had to accordingly change the padding inside the "Select all drafts" button to make sure the checkbox inside it aligns with the checkboxes in the drafts.



| 15px | 10px |
|--------|--------|
| ![image](https://github.com/zulip/zulip/assets/35286603/4d42b265-2299-4621-b63b-e4acbbacea05) | ![image](https://github.com/zulip/zulip/assets/35286603/d3609cf3-9440-4fdd-a905-5d7866b4b18b) | 


I think since the size of the checkbox is ~13px and constraining it inside `10px` resulted in the checkbox crossing the lines.

Reply:
Nice! It'd be good to record the reasoning behind the 15px width in a short comment or two in your CSS itself, so future contributors understand what's happening there (or how to translate things if we change those icons into a gridded presentation, rather than flexbox)